### PR TITLE
LCH-6901-3.6.x | Fix for ach-cfd command.

### PR DESCRIPTION
**Pull request by kirtigarg2584**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

File: src/ContentHubClient.php [hunk 1]
```diff
@@ -1334,8 +1334,7 @@ public function addFilterToWebhook($filter_id, $webhook_id) {
    * @throws \Exception
    */
   public function removeFilterFromWebhook($filter_id, $webhook_id) {
-    $options = ['body' => json_encode(['filter_id' => $filter_id])];
-    $response = $this->delete("settings/webhooks/$webhook_id/filters", $options);
+    $response = $this->delete("filters/$filter_id/subscriptions/$webhook_id");
```
Comment (narendradesai) [inline src/ContentHubClient.php:1337]: As per documentation here https://dev.acquia.com/api-documentation/acquia-content-hub-api#tag/filters/operation/PUT_v2-filters-uuid-subscriptions
`filters/$filter_id/subscriptions/$webhook_id` should be `filters/$filter_id/subscriptions/$subscriptionID`

OR is this a new endpoint which is not listed in documentation?

**Review by narendradesai — CHANGES_REQUESTED** [review]

File: src/ContentHubClient.php (hunk 1 above)
Comment (kirtigarg2584) [reply src/ContentHubClient.php:1337]: I had a discussion with Neha regarding this, she said that in documentation `subscriptionID` and `webhook_id` are considered as same.

**Review by narendradesai — APPROVED** [review]
Looks good 👍🏻

**Comment by narendradesai** [comment]
This is a bug fix, hence changes should also go in 3.5.x branch.

**Review by saxumVermes — APPROVED** [review]

**Pull request merged**